packages/amazonq/.changes/next-release/Bug Fix-7386c56a-07b9-4c6e-98f2-4321ec9cea28.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fix bug when pasting text from Mynah card with inline code causes new line breaks to form around the inline code text"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog). Will "mynah" mean anything to customers? 

The changelog is a place to advertise new features and fixes. It's the primary "mini blog" that allows us to give visbility to improvements. It is read by vscode users.

Reply:
Good point! Updating...